<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomViewer</class>
 <widget class="QMainWindow" name="RoomViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoomViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QTableView" name="tvItems"/>
       <widget class="QPlainTextEdit" name="pteInfoItems"/>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QTableView" name="tvServiceItems"/>
       <widget class="QTableView" name="tvProblemSolutionItems"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Добавить</string>
    </property>
    <addaction name="mnuAddItem"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mnuAddItem">
   <property name="text">
    <string>Оборудование...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
